pkg/kubectl/.import-restrictions

```diff
@@ -8,9 +8,9 @@
                 "k8s.io/kubernetes/pkg/api/pod",
                 "k8s.io/kubernetes/pkg/api/ref",
                 "k8s.io/kubernetes/pkg/api/resource",
-                "k8s.io/kubernetes/pkg/api/service",
+                "k8s.io/kubernetes/pkg/apis/core/helper/service",
```

Comment:
Why?  Just tidiness?  The registry one made a  lot of sense, but this appears more spread out.  Are there more coming?

Reply:
It's a core api helper and should live in some core package. Midterm goal is to clean up pkg/api from any legacy api code, possibly declare pkg/api as a integration package only and maybe some day get rid of it.

Reply:
> It's a core api helper and should live in some core package. Midterm goal is to clean up pkg/api from any legacy api code, possibly declare pkg/api as a integration package only and maybe some day get rid of it.

Ok.  If we're going to the effort of moving them, why not `pkg/apihelpers/<group>` so changes don't require api reviewers approval (these aren't changing types) and we can see just how many we have.